src/components/popover/examples/popover.tsx

```diff
@@ -16,22 +16,27 @@ export class PopoverExample {
 
     public render() {
         return [
-            <limel-button
-                primary={true}
-                label="Click me!"
-                onClick={this.openPopover}
-            />,
             <limel-popover open={this.isOpen} onClose={this.onPopoverClose}>
+                <limel-button
+                    slot="trigger"
+                    primary={true}
+                    label="Click me!"
+                    onClick={this.openPopover}
+                />
```

Comment:
Note that this needs doing in the table too. (This would be a breaking change, if not for the fact that we haven't published the docs for this component yet.)

Reply:
It will still work with the old way of doing it in lime-crm-components so it is not a breaking change. However, portal will not get the correct dimension and therefore the portal's container will sometimes be misplaced.

Reply:
sounds like we need to make a new issue for it, if you guys haven't already :)

Reply:
It also screws up the alignment of the columns headers a little:
![image](https://user-images.githubusercontent.com/154725/108518911-69907580-72c9-11eb-85df-3377d19bda74.png)

(The reason being that there's both an icon, and a space for the trigger, instead of just the space for the trigger with the icon _inside_ it 😄

Reply:
Did either of you add an issue by the way? 😄

Reply:
Fixed in Lundalogik/lime-crm-components#339